anycloud/cli/src/deploy/mod.rs

```diff
@@ -993,7 +1008,19 @@ pub async fn new(
   }
   let resp = post_v1("new", body).await;
   let res = match resp {
-    Ok(res) => format!("Created App {} successfully!", style(res).bold()),
+    Ok(res) => {
+      // idc if it's been set before, I'm setting it now!!!
+      let _ = CLUSTER_ID.set(res);
```

Comment:
Does the `let _ =` is necessary?

Reply:
And, the `set` could be call without unwrapping? didn't know that.

Reply:
the `let _ =` is to prevent rust warnings on not using the `Result` - basically, if you `unwrap` the result of `set` and there was already a value, the cli will crash. if the `Result` is assigned to `_`, then it won't crash and won't cause a compiler warning :)